docs/networking.md

```diff
@@ -0,0 +1,114 @@
+<!--[metadata]>
++++
+title = "Networking in Compose"
+description = "How Compose sets up networking between containers"
+keywords = ["documentation, docs,  docker, compose, orchestration, containers, networking"]
+[menu.main]
+parent="smn_workw_compose"
+weight=6
++++
+<![end-metadata]-->
+
+
+# Networking in Compose
+
+By default, Compose sets up a single default [network](http://TODO/docker-networking-docs) for your app. Each container for a service joins the default network and is discoverable via DNS under the service's name.
+
+> **Note:** Your app's network is given the same name as the "project name", which is based on the name of the directory it lives in. See the [CLI docs](cli.md#p-project-name-name) for how to override it.
```

Comment:
Note that we need to be slightly careful of how we namespace network names since there is a reasonable chance they may end up featuring in service discovery namespacing at some point in the future.  e.g. myapp.mynetwork.mycompany.com as a DNS entry (with the DNS search path including .mynetwork.mycompany.com by default for myapp's containers, so a simple app doesn't need to worry about this namespacing).